components/event-bus/api/push/eventing.kyma-project.io/v1alpha1/types.go

```diff
@@ -63,13 +63,13 @@ type SubscriptionConditionType string
 type Conditions []SubscriptionCondition
 
 const (
-	// ConditionTrue for Kyma subscriptions.
+	// ConditionTrue value for the Kyma subscription condition status.
 	ConditionTrue ConditionStatus = "True"
-	// ConditionFalse for Kyma subscriptions.
+	// ConditionFalse value for the Kyma subscription condition status.
 	ConditionFalse ConditionStatus = "False"
 
-	// Ready for Kyma subscriptions.
+	// Ready label for the Kyma subscription condition type.
 	Ready SubscriptionConditionType = "is-ready"
-	// EventsActivated for Kyma subscriptions.
+	// EventsActivated label for the Kyma subscription condition type.
```

Comment:
@bszwarc Please advise, should this be `of` or `for`?

Reply:
I would say `for` :)

Reply:
@bszwarc Thanks!